libs/bixarena/api-client-python/bixarena_api_client/api/auth_api.py

```diff
@@ -0,0 +1,1215 @@
+# coding: utf-8
+
+"""
+BixArena AI API
+
+Advance bioinformatics by evaluating and ranking AI agents.
+
+The version of the OpenAPI document: 1.0.0
+Generated by OpenAPI Generator (https://openapi-generator.tech)
+
+Do not edit the class manually.
+"""  # noqa: E501
+
+import warnings
+from pydantic import validate_call, Field, StrictFloat, StrictStr, StrictInt
+from typing import Any, Dict, List, Optional, Tuple, Union
+from typing_extensions import Annotated
+
+from pydantic import StrictStr
+from bixarena_api_client.models.get_jwks200_response import GetJwks200Response
+from bixarena_api_client.models.mint_internal_token200_response import (
+    MintInternalToken200Response,
+)
+from bixarena_api_client.models.oidc_callback200_response import OidcCallback200Response
+
+from bixarena_api_client.api_client import ApiClient, RequestSerialized
+from bixarena_api_client.api_response import ApiResponse
+from bixarena_api_client.rest import RESTResponseType
+
+
+class AuthApi:
```

Comment:
We could try to use the Auth API provided by this client in the future. For now, the web app is implementing the redirection to start the OIDC auth flow and store the session cookie.